src/hooks/useENSRegistrationStepHandler.tsx

```diff
@@ -145,13 +160,24 @@ export default function useENSRegistrationStepHandler(observer = true) {
   useEffect(() => {
     if (observer) return;
     let interval: NodeJS.Timer;
-    if (registrationStep === REGISTRATION_STEPS.WAIT_ENS_COMMITMENT) {
+    if (
+      (registrationParameters.commitTransactionConfirmedAt ||
+        registrationStep === REGISTRATION_STEPS.WAIT_ENS_COMMITMENT) &&
+      !readyToRegister
```

Comment:
if `readyToRegister` we don't need to keep updating the seconds that have passed since the commit confirmation, this was running all the time before doing nothing but adding 1 to this time